src/sentry/models/monitorenvironment.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_only_model,
+    sane_repr,
+)
+from sentry.models.monitor import MonitorStatus
+
+
+@region_silo_only_model
+class MonitorEnvironment(Model):
```

Comment:
Could this (and monitors in general) be added to scheduled deletions so that when a project/organization are deleted the monitors are also cleaned up? It doesn't have to be in this pull request but it would be good to address before Monitors gets even more schema.

Reply:
I think Monitors are, but could be wrong. This is just a placeholder for now, but will make sure they are

Reply:
They aren't, just need to add them here: https://github.com/getsentry/sentry/blob/82f1ac4f6c9824fc5e37250755f9fa5717563868/src/sentry/deletions/defaults/project.py#L4